permafrost/templates/permafrost/permafrostrole_form.html

```diff
@@ -59,12 +70,21 @@ <h6 class="m-0 text-primary font-weight-bold">
                         {{ form.description.label }}
                     </label>
                     {% if form.description.required %}<span class="required">*</span>{% endif %}
+                    {% if object and object.is_default_role %}
```

Comment:
just add a class name and write the div to the html regardless - this way you'll only need one if statement